packages/ui/src/variables.css

```diff
@@ -30,7 +30,7 @@
     0 25.600000381469727px 57.599998474121094px 0 #00000038; /* Primarily used for dialogs */
   --surface-color-primary-button-default: #013366;
   --surface-color-primary-button-hover: #1e5189;
-  --surface-color-primary-button-disabled: #9f9d9c;
+  --surface-color-primary-button-disabled: #edebe9;
```

Comment:
Glad you did this. I used the axe site's tool to get as light a grey of the same hue as I could to pass the test but I guess their site is more conservative than the tests we run? idk

Reply:
Yeah I just used the WAVE chrome extension. It has a nice built in one on the page. But also the original one didn't fail via the tool, but did fail when inputting the values directly. 🤷🏻 

https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh